controllers/hibp.js

```diff
@@ -12,7 +12,16 @@ async function notify (req, res) {
   const subscribers = await DB.getSubscribersByHashes(hashes);
 
   const reqBreachName = req.body.breachName.toLowerCase();
-  const breach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === reqBreachName);
+  let breach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === reqBreachName);
+  if (!breach) {
+    // If breach isn't found, try to reload breaches from HIBP
+    await HIBP.loadBreachesIntoApp(req.app);
```

Comment:
Are we throttling this somehow? Like, if I call some API endpoint with a bad breach name and cause it to scan our local breaches and then hit Troy's endpoints, but then do that at scale...

Or did we have some code in place to make sure we aren't hitting HIBP more than once per every 5 minutes?